packages/block-editor/src/components/image-link-destinations/index.native.js

```diff
@@ -11,13 +11,12 @@ import { __ } from '@wordpress/i18n';
 import { Icon, check, chevronRight } from '@wordpress/icons';
 import { blockSettingsScreens } from '@wordpress/block-editor';
```

Comment:
My perception is that the project generally prefers importing internal dependencies. We could change this to a local import now that this file resides within the `block-editor` package.